centreon/packages/ui/src/Listing/Cell/DataCell.tsx

```diff
@@ -114,11 +125,15 @@ const DataCell = ({
           }}
           {...commonCellProps}
         >
-          <Component
-            isHovered={isRowHovered}
-            isSelected={isRowSelected}
-            row={row}
-          />
+          <>
+            <InvisibleIcon className={classes.componentColumn} />
```

Comment:
You don't need a width here as it's already defined in the SvgIcon component

Reply:
i override the width to 18px , because columns of type component seem like there are not aligned with text header 🤔 with no override : 
![noOverride](https://user-images.githubusercontent.com/97687698/208448983-859164fe-c643-442d-a5fb-7f29d1cac8b0.PNG)